nengo/builder/learning_rules.py

```diff
@@ -83,6 +83,80 @@ def step_simoja():
         return step_simoja
 
 
+class SimVoja(Operator):
+    """Simulates a simplified version of Oja's rule in the vector space.
```

Comment:
We typically don't have detailed docstrings like this for operators, since users aren't likely to see them. I'm wondering if there's a better place for some of this stuff, especially the little derivation. The notebook, perhaps?